### ISSUE TITLE
Plugin App17 cannot fetch any stream

### ISSUE THREAD

**Issue by Tony-Liou** [streamlink/streamlink#2970]
## Plugin Issue

- [x] This is a plugin issue and I have read the contribution guidelines.

### Description

`Stream currently unavailable.`
The output of the Streamlink is always the same. Even if the streamer is streaming online.

### Reproduction steps / Explicit stream URLs to test

Paste any URL (e.g., `https://17.live/live/{any stream ID}`) and execute the command.

### Log output

```
C:\>streamlink https://17.live/live/7000362 -l trace
[22:44:25,408][cli][debug] OS:         Windows 10
[22:44:25,409][cli][debug] Python:     3.6.6
[22:44:25,410][cli][debug] Streamlink: 1.4.1
[22:44:25,410][cli][debug] Requests(2.23.0), Socks(1.7.1), Websocket(0.57.0)
[22:44:25,410][cli][info] Found matching plugin app17 for URL https://17.live/live/7000362
[22:44:25,735][plugin.app17][info] Stream currently unavailable.
error: No playable streams found on this URL: https://17.live/live/7000362
```

### Additional comments, screenshots, etc.

[screenshot](https://imgur.com/MFn7K0y)

**Comment by bastimeyer** [comment]
There's already a PR with a fix for that which you could try out: #2969

**Comment by gravyboat** [comment]
Closing since there's already an open PR.
